stdlib/3/multiprocessing/connection.pyi

```diff
@@ -0,0 +1,14 @@
+from typing import Any
```

Comment:
This file seems to have a few more public classes (https://github.com/python/cpython/blob/master/Lib/multiprocessing/connection.py#L10). I'm going to try to write stubs for those myself.